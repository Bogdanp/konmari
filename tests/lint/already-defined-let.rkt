#lang racket/base

(let ([x 5]
      [x 42])
  x)

(let loop ([x 5]
           [x 42])
  x)

(define (f x)
  (let ([x 5]
        [x 10])
    x))
