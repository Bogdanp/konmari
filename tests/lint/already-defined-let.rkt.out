/already-defined-let.rkt:4:8:error:identifier 'x' is already defined
/already-defined-let.rkt:8:13:error:identifier 'x' is already defined
/already-defined-let.rkt:13:10:error:identifier 'x' is already defined
